.github/ISSUE_TEMPLATE.md

```diff
@@ -1,53 +1,30 @@
-<!-- Thanks for filing an issue! Before hitting the button, please answer these questions.-->
+<!-- This form is for bug reports and feature requests ONLY! 
 
-**Is this a request for help?** (If yes, you should use our troubleshooting guide and community support channels, see https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/.):
-
-**Note:** Please file issues for subcomponents under the appropriate repo
-
-| Component | Repo                                                               |
-| --------- | ------------------------------------------------------------------ |
-| kubectl   | [kubernetes/kubectl](https://github.com/kubernetes/kubectl/issues/new) |
-| kubeadm   | [kubernetes/kubeadm](https://github.com/kubernetes/kubeadm/issues/new) |
-
-**What keywords did you search in Kubernetes issues before filing this one?** (If you have found any duplicates, you should instead reply there.):
-
----
-
-**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
-
-<!--
-If this is a BUG REPORT, please:
-  - Fill in as much of the template below as you can.  If you leave out
-    information, we can't help you as well.
-
-If this is a FEATURE REQUEST, please:
-  - Describe *in detail* the feature/behavior/change you'd like to see.
-
-In both cases, be ready for followup questions, and please respond in a timely
-manner.  If we can't reproduce a bug or think a feature already exists, we
-might close your issue.  If we're wrong, PLEASE feel free to reopen it and
-explain why.
+If you're looking for help check [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes) and the [troubleshooting guide](https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/).
 -->
 
-**Kubernetes version** (use `kubectl version`):
-
-
-**Environment**:
-- **Cloud provider or hardware configuration**:
-- **OS** (e.g. from /etc/os-release):
-- **Kernel** (e.g. `uname -a`):
-- **Install tools**:
-- **Others**:
+Is this a BUG REPORT or FEATURE REQUEST?:
 
+<!-- Uncomment only one, leave it on its own line: 
```

Comment:
Hmm.. you're right. Maybe a `>` ? That would block quote it. Or single ticks? But then there would be two characters that would have to be removed. IIRC, the bot only cares if it starts the line with `/`